articles/embedding-a-video-into-a-webpage-using-html-and-css/Index.md

```diff
@@ -0,0 +1,139 @@
+
+**Step 5:** Click on the copy. 
+As shown in the figure below, click on the copy and paste the code in your HTML page(assuming you have the HTML page/code already but if not, do not worry, the steps will be shown afterwards 😊).
+
+ ![step5](/engineering-education/embedding-a-video-into-a-webpage-using-html-and-css/step-5.jpg)                         
+
+In our case, the code/iframe tag looks like this:
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+<title>Put your preferred title</title>
+<style>
+
+body{ background-color: purple;
+background-text: white;}
+
+</style>
+</head>
+<body>
+
+<h1>EMBEDDING A VIDEO</h1>
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/t4vKPhjcMZg" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+</body>
+</html>
+```
+
+**Step 6:** Open your HTML page.
+Locate your HTML page, where you want to paste the iframe tag. 
+
+**Step 7:** Type or paste the iframe tag into your HTML page that you copied from YouTube. Below is a sample of a simple HTML page used for demonstration. 
+
+![step 7](/engineering-education/embedding-a-video-into-a-webpage-using-html-and-css/step-7.jpg)
+
+**Step 8:** Save the changes you made on your HTML page. 
+After saving, while you are still on your HTML page, right click and select “Open in Browser”. By clicking on Open in Browser, your HMTL code will run in your browser.
+
+![step 8](/engineering-education/embedding-a-video-into-a-webpage-using-html-and-css/step-8.jpg)
+
+**Step 9:** Your Browser’s output.
+Depending on your HTML code(on how you have styled it), you can place the iframe tag anywhere you want within your HTML page. After you have successfully embedded a video in your webpage, you can actively interact with it meaning, you can fast forward and rewind it, play and pause, increase, and decrease the volume or even mute, click on YouTube icon in the embedded page to play the video directly on YouTube etc. All those activities are called video controls.
+
+![step 9](/engineering-education/embedding-a-video-into-a-webpage-using-html-and-css/step-9.jpg)
+
+**Step 10:** Enjoy your view.
+As mentioned earlier, CSS is basically for styling your webpage. Like the above screenshot contains a brown-ish/red-ish background and the screenshot below contains a purple background. You can play with the HTML and CSS skills to suit your preferences.
+ 
+ ![step 10](/engineering-education/embedding-a-video-into-a-webpage-using-html-and-css/step-10.jpg)
+
+### Important Tips
+- Avoid hyperlinked videos: The URL’s Video should be on its own without any characters or spaces before or after the URL and should not be a hyperlinked or it will not be embedded. What is hyperlink? is a highlight done to a link such that when it is clicked or hovered, the browser automatically displays another page or changes the current page to show the referenced content. 
```

Comment:
The URL’s Video should be on its own without any characters or spaces before or after the URL and should not be a hyperlinked or it will not be embedded.

Rephrase this sentence.